quarkus/server/build.gradle.kts

```diff
@@ -68,7 +68,12 @@ tasks.named("distTar") { dependsOn("quarkusBuild") }
 tasks.register("run") { dependsOn("quarkusRun") }
 
 tasks.named<QuarkusRun>("quarkusRun") {
-  jvmArgs = listOf("-Dpolaris.bootstrap.credentials=POLARIS,root,secret")
+  jvmArgs =
+    listOf(
+      "-Dpolaris.bootstrap.credentials=POLARIS,root,secret",
+      "-Dquarkus.console.color=true",
+      "-Dquarkus.log.file.enable=false",
```

Comment:
Log file would be helpful in case of debugging.

Reply:
Thought about that too, but do you really think people would need to debug these runs _after the fact_? I think the console logs are enough.

Reply:
It would be helpful for debugging in the following cases:
1. Local integration tests. 
2. Multi-steps operations are involved. 

I would keep it if it doesn't bother us too much. Does it?

Reply:
> The other problem with file logging is that they tend to pile up and you could end up with tons of .tgz rotated files in the ./logs directory.

This is a reasonable complaint, can we make it use `/tmp` somehow?

Reply:
I will just use `rm -rf ./logs` to clean it up. It doesn't bother me too much.

Reply:
Just my $.02, but I have never ever used the log files

Reply:
Thanks all for the feedback, I am starting to wonder whether the cure is worse than the disease, IOW, maybe the best thing to do here is to not do anything.

Each developer can handle this in their own ways, e.g. by removing the log files periodically as suggested by @flyrain, or by just appending `-Dquarkus.log.file.enable=false` manually to `./gradlew run` (which is probably what I would do).

So, if you all agree, let's just close this PR without merging it.

Reply:
Thanks @adutra for keeping working on it. Can we still keep the color change?

Reply:
Oh sure, that's also an option. Let me update the PR then.